dlme_airflow/utils/dataframe.py

```diff
@@ -10,6 +10,26 @@ def datafile_for_collection(collection):
     return working_csv
 
 
+def datafile_for_collection_archive(collection):
+    working_csv = os.path.join(
+        os.path.abspath("working"), collection.data_path(), "archive", "data.csv"
```

Comment:
Doesn't `data.csv` need a timestamp in it, e.g. `data-202203151340121.csv`? I  had to do something like [this](https://github.com/sul-dlss/dlme-airflow/blob/main/dlme_airflow/tasks/archive.py#L109-L119) to get the latest archive file. I almost found myself wanting to put this function or something like it on the Collection class.